lib/github-pages-health-check/domain.rb

```diff
@@ -21,11 +21,11 @@ class Domain < Checkable
 
       HASH_METHODS = [
         :host, :uri, :dns_resolves?, :proxied?, :cloudflare_ip?, :fastly_ip?,
-        :old_ip_address?, :a_record?, :cname_record?, :valid_domain?,
-        :apex_domain?, :should_be_a_record?, :cname_to_github_user_domain?,
-        :cname_to_pages_dot_github_dot_com?, :cname_to_fastly?,
-        :pointed_to_github_pages_ip?, :pages_domain?, :served_by_pages?,
-        :valid_domain?
+        :old_ip_address?, :a_record?, :cname_record?, :has_mx_records?,
```

Comment:
Is the `has_` necessary? isn't that a Ruby style thing (that `has_` is implied by `?`)? Or at the very least, to mirror the others?